pages/thc/[processRef]/manager-review.tsx

```diff
@@ -0,0 +1,314 @@
+
+  const outsidePropertyImages = useDataValue(
+    Storage.ProcessContext,
+    "property",
+    processRef,
+    (values) => (processRef ? values[processRef]?.outside : undefined)
+  );
+
+  const outsidePropertyImageThumbnails = outsidePropertyImages.result
+    ? outsidePropertyImages.result.images.map((image) => (
+        <Thumbnail
+          key={image}
+          src={image}
+          alt={"Thumbnail of an uploaded image"}
+        />
+      ))
+    : [];
+
+  const extraRows = [
+    { key: "Outside property", value: outsidePropertyImageThumbnails },
+    { key: "Completed by", value: officerFullNameValue },
+    { key: "Date completed", value: submittedDateValue },
+    { key: "The visit was", value: visitType },
+    { key: "The visit took place", value: visitTookPlace },
+    {
+      key: "Present for check",
+      value:
+        presentTenantNames.length > 0
+          ? presentTenantNames.join(", ")
+          : "Loading...",
+    },
+  ].filter(({ value }) => value.length > 0);
+
+  return (
+    <MainLayout
+      title={PageTitles.ManagerReview}
+      heading="Review Tenancy and Household Check"
+    >
+      <React.Fragment>
+        <TenancySummary
+          details={{
+            address: address.result,
+            tenants: allTenantNames,
+            tenureType: tenureType.result,
+            startDate: tenancyStartDate.result,
+          }}
+          extraRows={extraRows}
+        />
+      </React.Fragment>
+      <IdAndResidencyReviewSection
+        tenants={tenantsWithPresentStatus}
+        readOnly
+      />
+      <HouseholdReviewSection readOnly />
+      <PropertyInspectionReviewSection readOnly />
+      <WellbeingSupportReviewSection readOnly />
+      {otherNotesValues.length > 0 && (
+        <>
+          <Heading level={HeadingLevels.H2}>Other Notes</Heading>
+          {otherNotesValues.map((note, index) => (
+            <Paragraph key={index}>{note}</Paragraph>
+          ))}
+        </>
+      )}
+      <Heading level={HeadingLevels.H2}>Declaration</Heading>
+      <Paragraph>
+        I confirm that the information I have given for the purposes of this
+        form is true and accurate. I understand that giving false information
+        may amount to fraud and would put my tenancy at risk with the result
+        that I may lose my home.
+      </Paragraph>
+      <Paragraph>Date of visit: {submittedDateValue}</Paragraph>
+      {tenantsPresent.map(
+        ({ fullName, id }) =>
+          signatureValues[id] && (
+            <React.Fragment key={id}>
+              <Heading level={HeadingLevels.H3}>
+                {fullName}&apos;s signature
+              </Heading>
+              <img className="signature" src={signatureValues[id]} />
+            </React.Fragment>
+          )
+      )}
+      <Heading level={HeadingLevels.H2}>Manager&apos;s comment</Heading>
+      <Textarea
+        name="manager-comment"
+        label={{ children: "" }}
```

Comment:
Let's use `<Heading level={HeadingLevels.H2}>Manager&apos;s comment</Heading>` as the label here instead of as a separate component.